extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -304,7 +305,8 @@ export class SubscriptionService {
         this.pageConfig_.getProductId(),
         'Product id is null'
     ));
-    this.platformStore_ = new PlatformStore(serviceIds);
+    this.platformStore_ = new PlatformStore(serviceIds,
```

Comment:
Few lines here are clear duplicates of non-delegated case. See if you can factor it out.

Reply:
This is no more the same case after last PR